src/en/guide/plugins/creatingAndInstallingPlugins.adoc

```diff
@@ -276,7 +276,7 @@ Although the link:../ref/Command%20Line/create-plugin.html[create-plugin] comman
 * `grails-app/build.gradle` (although it is used to generate `dependencies.groovy`)
 * `grails-app/conf/application.yml` (renamed to plugin.yml)
 * `grails-app/conf/spring/resources.groovy`
-* `grails-app/conf/logback.groovy`
+* `grails-app/conf/logback-spring.xml`
```

Comment:
`logback.xml` and `application.groovy` are also be excluded

Reply:
To your knowledge will `application.groovy` be renamed to `plugin.groovy` as well?

Reply:
Also, my opinion is to not document `logback.xml` if we decided to go with `logback-spring.xml`

Reply:
> To your knowledge will application.groovy be renamed to plugin.groovy as well?

To my knowledge, it does not get automatically renamed. That comment is a bit cryptic. To my knowledge, if a plugin author wants config from the plugin to be included in consuming applications, he/she should put it in a file named `plugin.yml`.

> Also, my opinion is to not document logback.xml if we decided to go with logback-spring.xml

My opinion is to document both, for the time being. Otherwise, people might think they have to switch.

Reply:
Cheking the Spring Boot docs, both names can be used (preferring `logback-spring.yml`) https://docs.spring.io/spring-boot/reference/features/logging.html#features.logging.custom-log-configuration

Why not just documenting `logback-spring.xml` adding a link to the Spring documentation letting people see all the quirks and quarks of configuring the logs?

On a side note, I just found out that we can also configure the logs directly in the `application.yml` it may be a good option so we can remove the `logback-spring.xml` file altogether (I'll give this a try and maybe open a feature request).

Reply:
> Why not just documenting logback-spring.xml

Well, `logback.xml` is in fact excluded and that should be documented.

Reply:
You are telling me two different things, which one do we want to pursue?

@matrei > "I think we could drop all other mentions of logback.xml in the docs."
@matrei > "logback.xml is in fact excluded and that should be documented."

I would like to remove all the instances of `logback.xml` except in the introductiory chapter where we talk about it as a legacy feature.

Reply:
Well, in this particular place, I think both should be included.